packages/react/src/preview-card/positioner/usePreviewCardPositioner.ts

```diff
@@ -97,7 +102,7 @@ export namespace usePreviewCardPositioner {
      * The side of the anchor element that the preview card element should align to.
      * @default 'bottom'
      */
-    side?: Side;
+    side?: Side | InlineSide;
```

Comment:
Nit: can we use a single type instead of a union? Should be slightly easier to navigate the types for the developer

Reply:
Updated!